benefit/data/sheets/CaseEvent/CaseEvent.json

```diff
@@ -550,6 +550,20 @@
     "PostConditionState": "*",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/09/2021",
+    "CaseTypeID": "Benefit",
+    "ID": "actionPostponementRequest",
+    "Name": "Action Postponement Request",
+    "Description": "Action Postponement Request",
+    "DisplayOrder": 48,
```

Comment:
The next event is also in order 48 and has * for pre and post condition. When this event is enabled, have you tested that the other 48 order event is also visible. Please confirm

Reply:
It works fine